bugbug/db.py

```diff
@@ -115,13 +115,20 @@ def upload(path):
 
 
 def last_modified(path):
-    url = DATABASES[path]["url"]
-    last_modified = utils.get_last_modified(url)
 
-    if last_modified is None:
+    if is_different_schema(path):
         raise Exception("Last-Modified is not available")
 
-    return last_modified
+    else:
```

Comment:
You can leave the rest of the code as it was, no need to add an `else` branch (when we enter in the `if` branch, we raise, so we never execute anything else).

Reply:
Ok

Reply:
Done